index.html

```diff
@@ -161,6 +163,31 @@ <h3 class="inner">skills</h3>
             </ul>
         </div>
     </div>
+</section>
+<section class="about-me">
```

Comment:
- [ ] Kindly make sure you stick to the design as much as possible.
![image](https://user-images.githubusercontent.com/97100091/193920557-551a8c88-02e2-49fa-8a1d-8f1952e50ebe.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193920784-94573022-4c84-448a-a853-db68a57b98a8.png)